src/tests/Components/VideoItem.test.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { VideoItem } from '../../components/Home/VideoItem';
+import { videoData } from '../../__mock__/videoData';
+
+const { items } = videoData;
+
+console.log(items[1]);
+
+beforeEach(() => {
+  render(<VideoItem snippet={items[1].snippet} />);
+});
+
+describe('VideoItem elements', () => {
+  test('Should contain a channel title', () => {
+    const ChannelTitleElement = screen.queryByText(items[1].snippet.channelTitle);

```

Comment:
In this case, as you want to be in the document what you need is to use the `getByText`. Using `query*` won't rase an exception, so it will always pass. See: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence